src/utils.ts

```diff
@@ -134,3 +134,31 @@ export const DBOSJSON = {
 export function exhaustiveCheckGuard(_: never): never {
   throw new Error('Exaustive matching is not applied');
 }
+
+// Capture original functions
+const originalStdoutWrite = process.stdout.write.bind(process.stdout);
```

Comment:
Does this do anything by itself? Could it be pulled into the function?

Reply:
I think it is better to keep it here to capture the unmodified write functions at startup.

This logic gets the unmodified write, so the path would be:

`user defined write -> our intercept -> original stream`

if we capture the stream when setting up the intercept, we'd do:

`our intercept -> user defined write -> original stream`

so for example if the user enrich their logs, we'd lose that information in our export.


p.s.: this is not 100% fail proof, I believe if another file globally overwrites the writes before we reach this file, we'll save whatever was overriden

Reply:
Could this interfere with something else?  That's the big thing I'm worried about--I want this PR to do nothing unless OTLP is enabled.

Reply:
No interference from just saving the bound functions.